src/plugins/timeConductor/ConductorAxis.vue

```diff
@@ -207,9 +207,9 @@ export default {
             this.$emit('panAxis', panBounds);
         },
         endPan() {
-            const panBounds = this.dragStartX && this.dragX && this.dragStartX !== this.dragX
-                ? this.getPanBounds()
-                : undefined;
+            const panBounds = this.dragStartX && this.dragX && this.dragStartX !== this.dragX ?
```

Comment:
this formatting feel weird to me, but are we going forward with this? (might take some time for me to get used to it :P )

Reply:
I am personally more accustomed to `after`, and it is the default configuration for eslint which is why it was set to `after`. After reading a few examples online, I think `before` is actually more readable though (and AirBnB use `before`).

I really want to get this merged ASAP, so in this case I am going to switch to `before`, and let's see how we go, with an exception for `=`. I don't think `=` should ever start a line?

Reply:
I like `after` but maybe that's just me. I can see the argument for `before`. It doesn't seem logically consistent, though, to make an exception for `=`.

Reply:
I think this will be obsoleted anyways @nikhilmandlik . #3246